components/Sidebar/index.jsx

```diff
@@ -0,0 +1,93 @@
+import styles from './sidebar.module.css';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+
+const sidebarMenuOptions = [
+  {
+    name: 'Home',
+    iconPath: '/assets/Home.svg',
+    urlPath: '/',
+  },
+  {
+    name: 'Dashboard',
+    iconPath: '/assets/Dashboard.svg',
+    urlPath: 'dashboard',
+  },
+  {
+    name: 'Curreny Exchange',
+    iconPath: '/assets/Swap.svg',
+    urlPath: 'currency-exchange',
+  },
+  {
+    name: 'Auction',
+    iconPath: '/assets/Auction.svg',
+    urlPath: 'auction',
+  },
+  {
+    name: 'Stocks',
+    iconPath: '/assets/Stocks.svg',
+    urlPath: 'trading',
+  },
+  {
+    name: 'Balance',
+    iconPath: '/assets/Balance.svg',
+    urlPath: 'wallet',
+  },
+];
+
+const Sidebar = () => {
+  const router = useRouter();
+  const RDS_LOGO = '/assets/Real-Dev-Squad1x.png';
```

Comment:
Directly use the link I think there is no need to first assign it to a variable and then use it. 

as the logo will never be changed or the chances of changing the logo are less.